cypress-tests/cypress/support/pageBuilder/pbCreateCategory.ts

```diff
@@ -0,0 +1,48 @@
+import { gqlClient } from "../utils";
+
+declare global {
+    // eslint-disable-next-line @typescript-eslint/no-namespace
+    namespace Cypress {
+        interface Chainable {
+            pbCreateCategory(data: {
+                name: string;
+                slug: string;
+                layout: string;
+                url: string;
+            }): Promise<{ slug: string; name: string; layout: string; url: string }>;
+        }
+    }
+}
+
+const MUTATION = /* GraphQL */ `
+    mutation pbCreateCategory($data: PbCategoryInput!) {
+        pageBuilder {
+            category: createCategory(data: $data) {
+                data {
+                    slug
+                    name
+                    layout
+                    url
+                }
+            }
+        }
+    }
+`;
+
+Cypress.Commands.add("pbCreateCategory", data => {
+    return cy.login().then(user => {
+        return gqlClient
+            .request({
+                query: MUTATION,
+                variables: { data },
+                authToken: user.idToken.jwtToken
+            })
+            .then(response => {
+                if (response.pageBuilder.category.error) {
```

Comment:
This won't work because in L12, we're saying `pbCreateCategory` is returning `Promise<{ slug: string; name: string; layout: string; url: string }>;` but now, in case of an error, we're returning `response.pageBuilder.category.error`. So, the return type is no longer correct.

 I'd say don't bother with errors at all. We'll later think of a better way to handle these on a global level, so we don't have to do it in these GQL commands every time.

So, just do `return response.pageBuilder.category.data;` in L41 and that's it.